### ISSUE TITLE
SVG Loading

### ISSUE THREAD

**Issue by osmmartel** [PhenX/php-svg-lib#40]
Since PHP 7.2 (probably 7+), getting this error when loading SVGs:

**Error:**
count(): Parameter must be an array or an object that implements Countable
on line 259 in lib/php-svg-lib/src/Svg/Document.php

My code to implement an SVG in DomPDF:
`<img src="image/certificat_adhesion/top_ribbon_portrait.svg" id="top_ribbon">`

Image show up in PDF, but creates php error log entries due to error.

**This is the problem:**
Line 259:
`if (count($this->attributes)) {`

**Should be:**

`if (count($attributes)) {`

**Comment by zacek** [comment]
very similar issue:

```
PHP Notice:  Undefined offset: 0 in .../phenx/php-svg-lib/src/Svg/Tag/AbstractTag.php on line 146                                                                                                                                   
PHP Warning:  count(): Parameter must be an array or an object that implements Countable in .../phenx/php-svg-lib/src/Svg/Tag/AbstractTag.php on line 146
```
caused by `preg_match_all` that doesn't match and
```
...
146   if (count($match[0])) {
...
```

**Comment by joaos-beyian** [comment]
on line 259 in lib/php-svg-lib/src/Svg/Document.php, adding the following check

...
259      if (**isset($this->attributes) &&** count($this->attributes)) {
...

before doing php _count_ check should be done on variable first.

**Comment by bsweeney** [comment]
This cause of the OP's issue was addressed in 4909031bdabdc802cf49912d631690cb44b0240d.

@zacek your issue is an array handling issue but is different so I opened a new issue to address it.
